### PR TITLE
wgengine/netstack: make userspace ping work when tailscaled has CAP_NET_RAW

### PR DESCRIPTION
**Pull request by bradfitz**
Updates #3710

### CODE REVIEW

File: wgengine/netstack/netstack_linux.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+package netstack
+
+import (
+	"os/exec"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
```
Comment (maisem) [inline wgengine/netstack/netstack_linux.go:13]: there is some weird behavior on DSM6 which led to having to special case DSM7
https://github.com/tailscale/tailscale/blob/main/wgengine/router/router_linux.go#L180-L189

Comment (maisem) [reply wgengine/netstack/netstack_linux.go:13]: https://github.com/tailscale/tailscale/issues/2922#issuecomment-946026420

Comment (bradfitz) [reply wgengine/netstack/netstack_linux.go:13]: Done; PTAL

Comment (bradfitz) [reply wgengine/netstack/netstack_linux.go:13]: Confirmed that my tailscaled runs as uid 0 on DSM6 and non-0 on DSM7

**Review by maisem — APPROVED** [review]

**Pull request merged**